### DISCUSSION TITLE
Make Unhandled Message logs appear in WARNING rather than DEBUG

### DISCUSSION THREAD

**Discussion by Aaronontheweb** [akkadotnet/akka.net#4981] [General]
Ran into some issues with an Akka.NET application where the thing wouldn't run due to unhandled messages, which I couldn't see because the default logging level is `INFO`. Should we make these events appear inside the Akka.NET log by default by raising the log level to `WARNING` or `INFO`?

**Comment by ismaelhamed** [comment]
IMO users will then have to treat all messages in all the actor's defined behaviors just to avoid the warning/info logs.

I've always understood that was the reason you have to call `base.Unhandled()` yourself if you really care about unhandled messages, otherwise they're not even considered by the system. Once you do that, and if DEBUG level is not an option, then you can always either override that actor's `Unhandled` method or subscribe to `UnhandleMessage`s in the event stream (which is basically what `akka.actor.debug.unhandled = on` does).

**Reply by Aaronontheweb** [reply]
That makes sense to me - better to let users opt-in to ones they care about vs. having to opt-out of all the ones they don't.
